perian/perian-core/index.js

```diff
@@ -0,0 +1,37 @@
+// @flow
+import * as Derive from './src/bip32/derive.js'
+import * as ExtendedKey from './src/bip32/extendedKey.js'
+import * as Paths from './src/bip44/paths.js'
+import * as HDKey from './src/bip44/hdKey.js'
+import * as KeyPair from './src/commons/keyPair.js'
+import * as Network from './src/commons/network.js'
+import * as Crypto from './src/utils/crypto.js'
+import * as Base from './src/utils/base.js'
+import * as Plugin from './src/utils/plugin.js'
+
+const Utils = { Crypto, Base, Plugin }
+const Commons = { KeyPair, Network }
+const Bip32 = { Derive, ExtendedKey }
+const Bip44 = { Paths, HDKey }
+
+Plugin.addPlugin([
+  '@perian/network-bitcoin',
```

Comment:
OMG, so many packages! Seriously, are we doing this? Wow. I don't know how I feel about this. I guess it's OK, just very... split.